.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.9.18
```

Comment:
python 3.9.x is as far as we can go without replacing the test framework ("nose") and rewriting tests.